express-delete/server.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(Object.values(grades));
+  return next();
+});
+app.listen(8080, () => console.log('Port 8080 is connected'));
```

Comment:
Though I'm not sure it matters, it's best to put the `listen` last. That way you set up all your middleware and routes, then start listening.